CHANGELOG.md

```diff
@@ -3,10 +3,19 @@
 BREAKING CHANGES
   * `client.grpc` defaults to `true` now instead of `false`. This is to make it
      harder to misconfigure Connect [[GH-282](https://github.com/hashicorp/consul-helm/pull/282)]
-     
+
      If you do not wish to enable gRPC for clients, set `client.grpc` to
      `false` in your local values file.
 
+  * `connectInject.centralConfig` defaults to `true` now instead of `false`. This is to make it
+     easier to configure Connect via `service-defaults` and other routing
+     config [[GH-302](https://github.com/hashicorp/consul-helm/pull/302)]
+     See https://www.consul.io/docs/agent/options.html#enable_central_service_config
```

Comment:
```suggestion
     See [Consul Docs](https://www.consul.io/docs/agent/options.html#enable_central_service_config).
```

Reply:
This will be converted to a link by github anyway and I thought it best to make it clear where users are being linked to.